### PR TITLE
Fix 53648

### PR DESCRIPTION
**Pull request by Jammer77**

### CODE REVIEW

File: main/src/addins/MacPlatform/MainToolbar/ButtonBar.cs [hunk 1]
```diff
@@ -198,6 +211,35 @@ void UpdateButton (IButtonBarButton button, int idx)
...
+				hasFocus = value;
+				RebuildSegments ();
+			}
+		}
+		uint focusedSegment = 0; 
+		public bool IncreaseFocusIndex()
+		{
+			bool result = true;
+			focusedSegment++;
+			if (this.buttons.Count () <= focusedSegment+1) {
```
Comment (Therzok) [inline main/src/addins/MacPlatform/MainToolbar/ButtonBar.cs]: Use `buttons.Count` instead of `buttons.Count()`.

Comment (Jammer77) [inline main/src/addins/MacPlatform/MainToolbar/ButtonBar.cs]: I'v done it.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/ViewCommandHandlers.cs [hunk 2]
```diff
@@ -36,6 +36,8 @@
 using MonoDevelop.Components.Commands;
 using MonoDevelop.Ide.Commands;
 using MonoDevelop.Ide.Editor;
+using MonoDevelop.Components.AtkCocoaHelper;
+using System.Linq;
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/ViewCommandHandlers.cs]: Why is Linq needed?

Comment (Jammer77) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui/ViewCommandHandlers.cs]: Solved!

**Pull request closed without merging**